src/ferui-components/tree-view/tests/all.spec.ts

```diff
@@ -0,0 +1,7 @@
+import TreeViewClient from './tree-view-client.spec';
```

Comment:
Please, do not create a test folder. We didn't do it for other components.
Also, you need to name your test files like the component file name you want to test, suffixed with `.spec.ts`:
i.e : If your component file is `my-component.ts` then the test file will be `my-component.spec.ts`.
The `all.spec.ts` file should be at the root of you component folder.

Reply:
Moved the test files and deleted the test folder